arelle/plugin/validate/ROS/rules/ros.py

```diff
@@ -368,13 +368,13 @@ def rule_ros19(
         for unit, count in unit_counts.items():
             if count > principal_currency_value_count:
                 modelObjects = principal_currency_facts
-                message = _("'PrincipalCurrencyUsedInBusinessReport' has a value of %(principal_currency_value)s, "
+                message = _("'PrincipalCurrencyUsedInBusinessReport' has a value of %(principal_currency_values)s, "
```

Comment:
This output seems worse ("has a value of {USD}" vs "has a value of USD").  Why this change?